gateway/mw_jwt.go

```diff
@@ -792,7 +792,10 @@ func (k *JWTMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Request, _
 	k.reportLoginFailure(tykId, r)
 	if err != nil {
 		logger.WithError(err).Error("JWT validation error")
-		return errors.New("Key not authorized:Unexpected signing method."), http.StatusForbidden
+		errorDetails := strings.Split(err.Error(), ":")
+		if errorDetails[0] == "Unexpected signing method" {
+			return errors.New("Key not authorized:" + errorDetails[0] + "."), http.StatusForbidden
```

Comment:
errors should start lower case and shouldn't have `.` at the end

Reply:
I do agree but here it's a bit weird, since we use this middleware and return errors like that to API response, I wish that we refectory all of it and add proper messages we are about to return to users but this is at the moment big refactor thing